src/memory/memory.c

```diff
@@ -382,7 +381,9 @@ bool memory_reset_hww(void)
     // Set a new noise static private key.
     rust_noise_generate_static_private_key(rust_util_bytes_mut(
         chunk.fields.noise_static_private_key, sizeof(chunk.fields.noise_static_private_key)));
-    bool res = _write_chunk(CHUNK_1, chunk.bytes);
+    if (!_write_chunk(CHUNK_1, chunk.bytes)) {
```

Comment:
I guess the original intention was to run the below BLE stuff even if this fails, so to preserve this, just assign `bool res1` and `bool res2` and return `res1 && res2`?